src/js/covidMap.js

```diff
@@ -221,12 +248,25 @@ fluid.defaults("fluid.covidMap.map", {
             func: "fluid.covidMap.extractPostcodes",
             args: ["{that}.model.rows", "{that}.options.fields.postcode"]
         },
-        // Selection and hover state
         isHospitalSelected: {
             target: "dom.hospitalPanel.visible",
             source: "selectedIndex",
             func: "fluid.isValue"
         },
+        onlyShowHospitalPanelOnMobile: {
+            target: "visiblePanelOnMobileFlags",
+            source: "selectedIndex",
+            func: x => {
+                if (!!x) {
```

Comment:
Is this really right? Seems like it will treat an index of 0 as a missing value. You should use the same check as https://github.com/inclusive-design/covid-data-monitor/blob/8422fd6f800c4365fe222c2b9e2c29ffc58bb40c/src/js/covidMap.js#L251 - or perhaps factor these both into a dedicated isHospitalShowing model flag that both definitions can use